src/APIdata.js

```diff
@@ -0,0 +1,40 @@
+const ApiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
```

Comment:
- [ ] Unfortunately, I am still not able to create new scores in your app and I can still see the same issues in the console. Please, kindly update your project in order to match the requirements :raised_hands:
![image](https://user-images.githubusercontent.com/81522435/146657888-adfb8e88-c260-4723-a60f-703555372d2e.png)